src/lib/containers/DirectDownload.tsx

```diff
@@ -141,8 +148,8 @@ const DirectDownload: React.FunctionComponent<DirectFileDownloadProps> = props =
   const getIcon = () => {
     if (isExternalFile) {
       return (
-        <button className={'btn-download-icon'}>
-          <a rel="noreferrer" href={externalURL} target="_blank">
+        <button className={'btn-download-icon'} onClick={onClickCallback}>
+          <a className="ignoreLink" rel="noreferrer" href={externalURL} target="_blank">
```

Comment:
Ignore link (do not underline), and also associate the optional callback in the wrapper button (which will receive the event as it bubbles up).